src/card/card-header.jsx

```diff
@@ -7,15 +7,18 @@ function getStyles(props, state) {
 
   return {
     root: {
-      height: 72,
+      height: 'auto',
```

Comment:
I agree, hard-coding the height to `72` sounds like a bad idea. My point is that 'auto' is the default style value for the height property.